test/framework/src/main/java/org/elasticsearch/datageneration/datasource/DefaultMappingParametersHandler.java

```diff
@@ -36,34 +36,27 @@ public DataSourceResponse.LeafMappingParametersGenerator handle(DataSourceReques
             return null;
         }
 
-        var map = commonMappingParameters();
```

Comment:
This `map` was previously shared between invocations of a lambda returned by this method. That led to generated mappings being very similar most of the time which is not what we want.